doc/Customization-guide/Customizable-system-components/Owner-verifiers/index.md

```diff
@@ -5,30 +5,31 @@ permalink: /:path/
 sort_idx: 40
 ---
 
-* [Configuration schema](#configuration-schema)
-* [User verifier implementation](#user-verifier-implementation)
-* [User verifier descriptor](#user-verifier-descriptor)
-* [User verifier provisioning](#user-verifier-provisioning)
+* TOC
+{:toc}
 
-To implement a custom user verifier, you need to complete the following steps.
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+To implement a custom owner verifier, you need to complete the following steps.
```

Comment:
please add a small introduction with a link to Programming-guide/Key-platform-features/Endpoint-ownership. Otherwise this is out of context.